### ISSUE TITLE
Delta Pro Integration broken FW 1.0.0.96

### ISSUE THREAD

**Issue by Roughman007** [vwt12eh8/hassio-ecoflow#54]
Hi, I just updated my Delta Pro to firmware **1.0.0.96**. and now the Integration with Home Assistant is not working. I tried to remove and re-add my Delta Pro in Home Assistant but it still doesn't work. Anyone else facing similar problems?

Thanks.

**Comment by Jenda987** [comment]
Hello, I have same experience with my Delta Pro after update to latest firmware 1.0.0.96 ... all entities looks unavailable
nping -c 0 --tcp -p 8055 xx.xx.xx.xx
returns
RCVD (12.2177s) TCP xx.xx.xx.xx:8055 > yy.yy.yy.yy.ppp **RA** ttl=255 id=2448 iplen=40  seq=0 win=5760
I am afraid that **RA** meanst that port 8055 is closed.

**Comment by skpow55** [comment]
Same issue here with my EF DP.  Nmap regular scan shows no ports open.  
Also tried powering off/on EF DP and reloading HA integration.  Looks 
like EcoFlow closed the port with latest update?


nmap xxx.xxx.x.xxx

Nmap scan report for xxx.xxx.x.xxx

Host is up (0.025s latency).

All 1000 scanned ports on xxx.xxx.x.xxx are in ignored states.

Not shown: 1000 closed tcp ports (reset)

MAC Address: xx:xx:xx:xx:xx:xx (Espressif)

Nmap done: 1 IP address (1 host up) scanned in 16.18 seconds


-------------------------------------------------------------------------

**Comment by vwt12eh8** [comment]
related #58

**Comment by GeorgWolter** [comment]
Same problem here. I sent an e-mail to EcoFlow support.

**Comment by GeorgWolter** [comment]
EcoFlow answered:
"I will put your feedback on the VOC (Voice of Customer). We will keep improving our products and services."

As a workaround i am using the EcoFlow Cloud API and Node Red with the HTTP REQUEST Node (GET).
This gets me these 4 entities from Node Red into Home Assistant:

- soc
- remainTime
- wattsOutSum
- wattsInSum

URL and credentials (appKey & secretKey) are available from EcoFlow support after sharing DELTA's serial number.

**Comment by hiwo64** [comment]
Hi,
I have the same issue. 
Can you please describe how you did the work a round. THX

**Comment by GeorgWolter** [comment]
> Hi,
> I have the same issue.
> Can you please describe how you did the work a round. THX

Yes, you are welcome!

Q1: Credentials received from EF support?
Q2: Node Red Integration for HA up & running?
In Node Red you need:
A) Inject Node
B) HTTP Request Node GET
C) Change Node
D) HA Entity Node
E) Debug Node(s)

I can share the flow when you confirm Q1/Q2.

**Comment by Ne0-Hack3r** [comment]
> Hi, I just updated my Delta Pro to firmware **1.0.0.96**. and now the Integration with Home Assistant is not working. I tried to remove and re-add my Delta Pro in Home Assistant but it still doesn't work. Anyone else facing similar problems?
> 
> Thanks.

Both my DPs were on .96 firmware before I connected them to HA using this integration... HA is still pulling data...

**Comment by GeorgWolter** [comment]
> Both my DPs were on .96 firmware before I connected them to HA using this integration... HA is still pulling data...

Hi, so that means you can't find any problem? You (HA) can read / write all entities?
I have just tried add this integration via HACS again.
But now it's hanging / cycling at the field where i have entered the DELTA's pingable local IP.

**Comment by PaulHamburg** [comment]
Hi, my DP was on 1.0.0.96 firmware long before I connected them to HA yesterday using this integration. HA is pulling all data. Can read / write all entities. Can't find any problem.

**Comment by GeorgWolter** [comment]
A new FW 1.0.0.96(6) is available. I will try the integration now again.

**Comment by PaulHamburg** [comment]
…good Luck, Georg 👍

**Comment by GeorgWolter** [comment]
> …good Luck, Georg 👍

No chance. My DELTA Pro is not found with it's IP.

**Comment by Ne0-Hack3r** [comment]
> > Both my DPs were on .96 firmware before I connected them to HA using this integration... HA is still pulling data...
> 
> Hi, so that means you can't find any problem? You (HA) can read / write all entities? I have just tried add this integration via HACS again. But now it's hanging / cycling at the field where i have entered the DELTA's pingable local IP.

Yes, data for entities for both DPs is still populating and I can change settings from HA and see that reflected in the native app... I have HA, DPs, and a NUC with Win 10 and Android emulation running EF native app all on the same isolated vLAN... I also have DHCP reservations for the DPs so they essentially have static IPs... I really hope the next firmware update doesn't close this off... I've made a lot of use of this integration in the couple months I've had it.

**Comment by michael5411** [comment]
We are using two DeltaPro for commercial purposes. One has an elder firmware release and is working properly. The other one has faulty release ...96 and it's cut off. But the iPhone-app is still working on both DPs.

We sent a support request to the german branch of EcoFlow to get a new release asap. 
We'll keep you informed.

**Comment by Ne0-Hack3r** [comment]
> We are using two DeltaPro for commercial purposes. One has an elder firmware release and is working properly. The other one has faulty release ...96 and it's cut off. But the iPhone-app is still working on both DPs.
> 
> We sent a support request to the german branch of EcoFlow to get a new release asap. We'll keep you informed.

Which firmware version are you using on the DP that is still communicating with HA?

**Comment by GeorgWolter** [comment]
Info on DELTA Pro:
There is a new FW 3.0.2.21 (WLAN) and according to EcoFlow's support team Port 8055 is open.
Tested this integration with my DELTA Pro and it works again 🙏😁

![IMG_2768](https://user-images.githubusercontent.com/61526104/216530195-40923966-818e-4071-8f30-38231fc17db0.jpeg)

**Comment by ALERTua** [comment]
fyi: EcoFlow has announced a beta test of local API for Delta Pro in their Facebook group:
https://www.facebook.com/groups/1149081025558751
